src/mgmt/srv.c

```diff
@@ -62,12 +62,25 @@ static struct daos_rpc_handler mgmt_handlers[] = {
 static void
 mgmt_drpc_handler(Drpc__Call *request, Drpc__Response **response)
 {
-	D_DEBUG(DB_MGMT, "call received by mgmt drpc handler\n");
+	Drpc__Response *resp;
+
+	D_ALLOC_PTR(resp);
+	if (resp == NULL) {
+		D_ERROR("Failed to allocate response\n");
+		return
```

Comment:
";" after "return"? This caused the build failures as well as, I guess, the static analysis failure.